src/eko/constants.py

```diff
@@ -20,7 +11,13 @@
 """the normalization of fundamental generators"""
 
 CA = float(NC)
-"""second Casimir constant in the adjoint representation"""
+"""second Casimir constant in the adjoint representation - defaults to :math:`N_C = 3"""
 
 CF = float((NC * NC - 1.0) / (2.0 * NC))
-"""second Casimir constant in the fundamental representation"""
+"""second Casimir constant in the fundamental representation - defaults to :math:`\frac{N_C^2-1}{2N_C} = 4/3"""
```

Comment:
Since there has always been written that these numbers are only the default, let's add a small function to update them consistently:

```py
def update_colors(nc):
    global NC, CA, CF
    NC = int(nc)
    CA = float(NC)
    CF = float((NC * NC - 1.0) / (2.0 * NC))
```

Reply:
I wonder if this function is ever user ... however, if you add the function, you also have to provide a unit test

Reply:
> I wonder if this function is ever user ...

It's just to allow to change consistently, since we provide them in full generality, it's silly not to provide such a simple function.
However, it's not used of course, the proper question is if it *will be* used, and this you never know (e.g. in principle QCD with one color and suitable charges should be QED, so you might use it for testing).

>

Reply:
@niclaurenti please also add a docstring